celery/worker/request.py

```diff
@@ -489,7 +489,7 @@ def on_failure(self, exc_info, send_failed_event=True, return_ok=False):
             raise MemoryError(f'Process got: {exc_info.exception}')
         elif isinstance(exc_info.exception, Reject):
             return self.reject(requeue=exc_info.exception.requeue)
-        elif isinstance(exc_info.exception, Ignore):
+        elif isinstance(exc_info.exception, (Ignore, Terminated)):
             return self.acknowledge()
```

Comment:
This is probably not the right behavior.
We should reject revoked messages so monitoring systems would catch that these tasks were revoked.

I thought that this is what this feature does.